data/sensors_recent.php

```diff
@@ -138,16 +152,18 @@ function node_button($id) {
 EOF;
 	}
 
+
 	function gateway_button($gw_id, $gw_data = null) {
 		global $gateway_descriptions;
 
-		if (array_key_exists($gw_id, $gateway_descriptions))
-			$gw = $gateway_descriptions[$gw_id];
+		if (isset($gw_data['alias']))
+			$gw = $gw_data['alias'];
 		else
 			$gw = $gw_id;
 		$gw_html = htmlspecialchars($gw);
 
-		$filter_url_html = htmlspecialchars('?gateways=' . urlencode($gw_id) . '&show_other_gateways=1');
+		#$filter_url_html = htmlspecialchars('?gateways=' . urlencode($gw_id) . '&show_other_gateways=1');
+		$filter_url_html = htmlspecialchars('?gateways=' . urlencode($gw_id) . '&show_other_gateways=0');
```

Comment:
Heb je show_other_gateways hier bewust op 0 gezet, of was dat alleen om te testen?